service/vspo-schedule/web/biome.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.5.3/schema.json",
+  "organizeImports": {
+    "enabled": false
+  },
```

Comment:
[imo]
From now on, it might be a good to add vcs settings to only format the changed parts.
https://biomejs.dev/ja/reference/configuration/#vcs
```json
 "vcs": {
    "enabled": true,
    "clientKind": "git",
    "defaultBranch": "main",
    "useIgnoreFile": true
 }
```
Run the command:
```
biome format --changed --write
```